helm/bashrc

```diff
@@ -0,0 +1,66 @@
+# If not running interactively, don't do anything
+case $- in
+    *i*) ;;
+      *) return;;
+esac
+
+shopt -s checkwinsize
+shopt -s globstar
+
+shopt -s histappend
+HISTCONTROL=ignoreboth
+HISTSIZE=1000
+HISTFILESIZE=20000
+
+# set a fancy prompt (non-color, unless we know we "want" color)
+case "$TERM" in
+    xterm-color|*-256color) color_prompt=yes;;
+esac
+
+if [[ -x /usr/bin/tput ]] && tput setaf 1 >&/dev/null; then
+	# We have color support; assume it's compliant with Ecma-48
+	# (ISO/IEC-6429). (Lack of such support is extremely rare, and such
+	# a case would tend to support setf rather than setaf.)
+	color_prompt=yes
+fi
+
+if [[ $color_prompt = yes ]]; then
+    PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+else
```

Comment:
Not going to read all of this carefully, assume it works to your satisfaction, and your first mover advantage means the bash is set up the way you like it!

Reply:
Heh; stayed with the default debian, basically.  *Not* specifically my preference, but what is, I'm assuming, a middle ground.